articles/azure-fluid-relay/how-tos/rotate-fluid-relay-access-keys.md

```diff
@@ -0,0 +1,82 @@
+---
+description: Learn how to rotate Azure Fluid Relay access keys
+title: Rotate Azure Fluid Relay access keys
+ms.date: 08/13/2024
+ms.service: azure-fluid
+ms.topic: reference
+---
+
+# How to rotate Fluid Relay Server access keys
+This article provides an overview of managing access keys (tenant keys) in Azure Fluid Relay. 
+
+## Primary / Secondary keys
+Customers use the access keys to sign access tokens used to access Azure Fluid Relay resources, and Azure Fluid Relay uses the keys to validate the tokens. 
+
+Two keys are associated with each AFR resource: a primary key and secondary key. The purpose of dual keys is to let you regenerate, or roll, keys, providing continuous access to your account and data. 
+
+## View your access keys
+
+### [Azure Portal](#tab/azure-portal)
+To see your access keys, on the left menu select **Settings**. Then, select **Access Keys**. Select the **Copy** button to copy the selected key.
```

Comment:
We need to specify how they get to this screen. If they only have to open the Azure portal, then the following will do it. But it there are more steps, please add them.

```suggestion
To see your access keys, open the Azure Portal and on the left menu select **Settings**. Then, select **Access Keys**. Select the **Copy** button to copy the selected key.
```